ops/testing.py

```diff
@@ -3204,6 +3204,44 @@ def add_layer(
                         s._merge(service)
                     else:
                         layer.services[name] = service
+            for name, check in layer_obj.checks.items():
```

Comment:
Nit: could you please add a blank line between each top-level `for` loop to visually separate them a bit?